Third-Person Sample Crashes (D3D11)

#### System Information
- Ogre Version: v1.12.8 
- Operating System / Platform: Win10
- RenderSystem: D3D11

#### Detailed description

I'm able to run the SampleBrowser and open the third-person character sample. I can walk around and jump in that sample and it looks fine. However, when I try to pull out the sword (with Q) I get this error:
![D3DError](https://user-images.githubusercontent.com/12706574/90636711-02e6ae00-e1f9-11ea-9f7c-8579aa3cbf5a.PNG)

When I click OK, the game closes.

My dxdiag stuff:

![image](https://user-images.githubusercontent.com/12706574/90638383-45a98580-e1fb-11ea-9646-d10f04e98255.png)

[DxDiag.txt](https://github.com/OGRECave/ogre/files/5096626/DxDiag.txt)

My OGRE Engine Rendering Setup:

![image](https://user-images.githubusercontent.com/12706574/90641204-f2393680-e1fe-11ea-8f94-980b8f3d99a2.png)


#### Ogre.log

```
09:31:18: Creating resource group General
09:31:18: Creating resource group OgreInternal
09:31:18: Creating resource group OgreAutodetect
09:31:18: SceneManagerFactory for type 'DefaultSceneManager' registered.
09:31:18: Registering ResourceManager for type Material
09:31:18: Registering ResourceManager for type Mesh
09:31:18: Registering ResourceManager for type Skeleton
09:31:18: MovableObjectFactory for type 'ParticleSystem' registered.
09:31:18: ArchiveFactory for archive type FileSystem registered.
09:31:18: ArchiveFactory for archive type Zip registered.
09:31:18: ArchiveFactory for archive type EmbeddedZip registered.
09:31:18: DDS codec registering
09:31:18: ETC codec registering
09:31:18: Registering ResourceManager for type HighLevelGpuProgram
09:31:18: Registering ResourceManager for type Compositor
09:31:18: MovableObjectFactory for type 'Entity' registered.
09:31:18: MovableObjectFactory for type 'Light' registered.
09:31:18: MovableObjectFactory for type 'BillboardSet' registered.
09:31:18: MovableObjectFactory for type 'ManualObject' registered.
09:31:18: MovableObjectFactory for type 'BillboardChain' registered.
09:31:18: MovableObjectFactory for type 'RibbonTrail' registered.
09:31:18: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\RenderSystem_Direct3D9.dll
09:31:18: Installing plugin: D3D9 RenderSystem
09:31:18: D3D9 : Direct3D9 Rendering Subsystem created.
09:31:18: D3D9: Driver Detection Starts
09:31:18: D3D9: Driver Detection Ends
09:31:18: Plugin successfully installed
09:31:18: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\RenderSystem_Direct3D11.dll
09:31:18: Installing plugin: D3D11 RenderSystem
09:31:18: D3D11: Direct3D11 Rendering Subsystem created.
09:31:18: D3D11: Driver Detection Starts
09:31:18: D3D11: "NVIDIA GeForce GTX 1650"
09:31:18: D3D11: "AMD Radeon(TM) RX Vega 10 Graphics"
09:31:18: D3D11: "Microsoft Basic Render Driver (software)"
09:31:18: D3D11: Driver Detection Ends
09:31:18: Plugin successfully installed
09:31:18: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\RenderSystem_GL.dll
09:31:18: Installing plugin: GL RenderSystem
09:31:18: OpenGL Rendering Subsystem created.
09:31:18: Plugin successfully installed
09:31:18: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\RenderSystem_GL3Plus.dll
09:31:18: Installing plugin: GL 3+ RenderSystem
09:31:18: OpenGL 3+ Rendering Subsystem created.
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_ParticleFX.dll
09:31:19: Installing plugin: ParticleFX
09:31:19: Particle Emitter Type 'Point' registered
09:31:19: Particle Emitter Type 'Box' registered
09:31:19: Particle Emitter Type 'Ellipsoid' registered
09:31:19: Particle Emitter Type 'Cylinder' registered
09:31:19: Particle Emitter Type 'Ring' registered
09:31:19: Particle Emitter Type 'HollowEllipsoid' registered
09:31:19: Particle Affector Type 'LinearForce' registered
09:31:19: Particle Affector Type 'ColourFader' registered
09:31:19: Particle Affector Type 'ColourFader2' registered
09:31:19: Particle Affector Type 'ColourImage' registered
09:31:19: Particle Affector Type 'ColourInterpolator' registered
09:31:19: Particle Affector Type 'Scaler' registered
09:31:19: Particle Affector Type 'Rotator' registered
09:31:19: Particle Affector Type 'DirectionRandomiser' registered
09:31:19: Particle Affector Type 'DeflectorPlane' registered
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_BSPSceneManager.dll
09:31:19: Installing plugin: BSP Scene Manager
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_CgProgramManager.dll
09:31:19: Installing plugin: Cg Program Manager
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Codec_STBI.dll
09:31:19: stb_image - v2.23 - public domain image loader
09:31:19: Supported formats: jpeg,jpg,png,bmp,psd,tga,gif,pic,ppm,pgm,hdr
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_PCZSceneManager.dll
09:31:19: Installing plugin: Portal Connected Zone Scene Manager
09:31:19: PCZone Factory Type 'ZoneType_Default' registered
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_OctreeZone.dll
09:31:19: Installing plugin: Octree Zone Factory
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_OctreeSceneManager.dll
09:31:19: Installing plugin: Octree Scene Manager
09:31:19: Plugin successfully installed
09:31:19: Loading library C:/Projects/NonTerra/build/OGRE/bin/release/.\Plugin_DotScene.dll
09:31:19: Installing plugin: DotScene Loader
09:31:19: Plugin successfully installed
09:31:19: *-*-* OGRE Initialising
09:31:19: *-*-* Version 1.12.8 (Rhagorthua)
09:31:19: OverlayElementFactory for type Panel registered.
09:31:19: OverlayElementFactory for type BorderPanel registered.
09:31:19: OverlayElementFactory for type TextArea registered.
09:31:19: Registering ResourceManager for type Font
09:31:59: D3D11: RenderSystem Option: Rendering Device = NVIDIA GeForce GTX 1650
09:33:12: CPU Identifier & Features
09:33:12: -------------------------
09:33:12:  *   CPU ID: AuthenticAMD: AMD Ryzen 7 3750H with Radeon Vega Mobile Gfx
09:33:12:  *          SSE: yes
09:33:12:  *         SSE2: yes
09:33:12:  *         SSE3: yes
09:33:12:  *        SSE41: no
09:33:12:  *        SSE42: no
09:33:12:  *          MMX: yes
09:33:12:  *       MMXEXT: yes
09:33:12:  *        3DNOW: no
09:33:12:  *     3DNOWEXT: no
09:33:12:  *         CMOV: yes
09:33:12:  *          TSC: yes
09:33:12:  *INVARIANT TSC: yes
09:33:12:  *          FPU: yes
09:33:12:  *          PRO: yes
09:33:12:  *           HT: no
09:33:12: -------------------------
09:33:12: D3D11: Subsystem Initialising
09:33:12: D3D11: Driver Detection Starts
09:33:12: D3D11: "NVIDIA GeForce GTX 1650"
09:33:12: D3D11: "AMD Radeon(TM) RX Vega 10 Graphics"
09:33:12: D3D11: "Microsoft Basic Render Driver (software)"
09:33:12: D3D11: Driver Detection Ends
09:33:12: D3D11: Requested "NVIDIA GeForce GTX 1650", selected "NVIDIA GeForce GTX 1650"
09:33:12: ***************************************
09:33:12: *** D3D11: Subsystem Initialized OK ***
09:33:12: ***************************************
09:33:12: RenderSystem::_createRenderWindow "OGRE Sample Browser", 800x600 windowed  miscParams: FSAA=1 colourDepth=32 externalWindowHandle=1050654 gamma=No monitorIndex=NVIDIA GeForce GTX 1650 vsync=Yes vsyncInterval=1 
09:33:12: D3D11: Created D3D11 Rendering Window 'OGRE Sample Browser' : 800x600, 32bpp
09:33:12: Registering ResourceManager for type Texture
09:33:12: Registering ResourceManager for type GpuProgram
09:33:12: RenderSystem capabilities
09:33:12: -------------------------
09:33:12: RenderSystem Name: Direct3D11 Rendering Subsystem
09:33:12: GPU Vendor: nvidia
09:33:12: Device Name: NVIDIA GeForce GTX 1650
09:33:12: Driver Version: 27.21.14.5206
09:33:12:  * Fixed function pipeline: no
09:33:12:  * 32-bit index buffers: yes
09:33:12:  * Hardware stencil buffer: yes
09:33:12:    - Stencil depth: 8
09:33:12:    - Two sided stencil support: yes
09:33:12:    - Wrap stencil values: yes
09:33:12:  * Vertex programs: yes
09:33:12:    - Number of floating-point constants: 512
09:33:12:    - Number of integer constants: 16
09:33:12:    - Number of boolean constants: 16
09:33:12:  * Fragment programs: yes
09:33:12:    - Number of floating-point constants: 512
09:33:12:    - Number of integer constants: 16
09:33:12:    - Number of boolean constants: 16
09:33:12:  * Geometry programs: yes
09:33:12:    - Number of floating-point constants: 512
09:33:12:    - Number of integer constants: 16
09:33:12:    - Number of boolean constants: 16
09:33:12:  * Tessellation Hull programs: yes
09:33:12:    - Number of floating-point constants: 512
09:33:12:    - Number of integer constants: 16
09:33:12:    - Number of boolean constants: 16
09:33:12:  * Tessellation Domain programs: yes
09:33:12:    - Number of floating-point constants: 512
09:33:12:    - Number of integer constants: 16
09:33:12:    - Number of boolean constants: 16
09:33:12:  * Compute programs: yes
09:33:12:    - Number of floating-point constants: 512
09:33:12:    - Number of integer constants: 16
09:33:12:    - Number of boolean constants: 16
09:33:12:  * Supported Shader Profiles: cs_4_0 cs_4_1 cs_5_0 ds_5_0 gs_4_0 gs_4_1 gs_5_0 hlsl hs_5_0 ps_2_0 ps_2_a ps_2_b ps_3_0 ps_4_0 ps_4_0_level_9_1 ps_4_0_level_9_3 ps_4_1 ps_5_0 vs_2_0 vs_2_a vs_3_0 vs_4_0 vs_4_0_level_9_1 vs_4_0_level_9_3 vs_4_1 vs_5_0
09:33:12:  * Read-back compiled shader: yes
09:33:12:  * Number of vertex attributes: 32
09:33:12:  * Textures
09:33:12:    - Number of texture units: 16
09:33:12:    - Floating point: yes
09:33:12:    - Non-power-of-two: yes
09:33:12:    - 1D textures: yes
09:33:12:    - 2D array textures: yes
09:33:12:    - 3D textures: yes
09:33:12:    - Anisotropic filtering: yes
09:33:12:  * Texture Compression: yes
09:33:12:    - DXT: yes
09:33:12:    - VTC: no
09:33:12:    - PVRTC: no
09:33:12:    - ATC: no
09:33:12:    - ETC1: no
09:33:12:    - ETC2: no
09:33:12:    - BC4/BC5: no
09:33:12:    - BC6H/BC7: yes
09:33:12:    - ASTC: no
09:33:12:    - Mipmaps for compressed formats: no
09:33:12:  * Vertex Buffers
09:33:12:    - VET_UBYTE4 element type: yes
09:33:12:    - Render to Vertex Buffer: yes
09:33:12:    - Instance Data: yes
09:33:12:    - Primitive Restart: yes
09:33:12:  * Vertex texture fetch: yes
09:33:12:    - Max vertex textures: 4
09:33:12:    - Vertex textures shared: yes
09:33:12:  * Read/Write Buffers: no
09:33:12:  * Hardware Occlusion Query: yes
09:33:12:  * User clip planes: yes
09:33:12:  * Infinite far plane projection: yes
09:33:12:  * Hardware render-to-texture: yes
09:33:12:    - Multiple Render Targets: 8
09:33:12:    - With different bit depths: yes
09:33:12:  * Point Sprites: no
09:33:12:  * Wide Lines: no
09:33:12:  * Hardware Gamma: no
09:33:12: DefaultWorkQueue('Root') initialising on thread 8544.
09:33:12: Particle Renderer Type 'billboard' registered
09:33:12: SceneManagerFactory for type 'BspSceneManager' registered.
09:33:12: Registering SceneLoader Q3BSP
09:33:12: DefaultWorkQueue('Root')::WorkerFunc - thread 16624 starting.
09:33:12: DefaultWorkQueue('Root')::WorkerFunc - thread 8244 starting.
09:33:12: SceneManagerFactory for type 'PCZSceneManager' registered.
09:33:12: MovableObjectFactory for type 'PCZLight' registered.
09:33:12: MovableObjectFactory for type 'Portal' registered.
09:33:12: MovableObjectFactory for type 'AntiPortal' registered.
09:33:12: PCZone Factory Type 'ZoneType_Octree' registered
09:33:12: SceneManagerFactory for type 'OctreeSceneManager' registered.
09:33:12: Registering SceneLoader DotScene
09:33:12: Creating resource group BSPWorld
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/oa_rpg3dm2.pk3' of type 'Zip' to resource group 'BSPWorld'
09:33:12: Creating resource group Essential
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/thumbnails' of type 'FileSystem' to resource group 'Essential'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/packs/SdkTrays.zip' of type 'Zip' to resource group 'Essential'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/packs/profiler.zip' of type 'Zip' to resource group 'Essential'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/fonts' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/programs' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/scripts' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/textures' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/textures/terrain' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/models' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/particle' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/DeferredShadingMedia' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/DeferredShadingMedia/DeferredShading/post' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/PCZAppMedia' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/scripts/SSAO' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/textures/SSAO' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/volumeTerrain' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/CSMShadows' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/PBR' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/PBR/filament' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/textures/PBR' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/HLMS' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/cubemap.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/cubemapsJS.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/dragon.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/fresneldemo.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/ogretestmap.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/ogredance.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/Sinbad.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/skybox.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/volumeTerrain/volumeTerrainBig.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/DamagedHelmet.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/packs/filament_shaders.zip' of type 'Zip' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/ShadowVolume' of type 'FileSystem' to resource group 'OgreInternal'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/RTShaderLib/materials' of type 'FileSystem' to resource group 'OgreInternal'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/RTShaderLib/GLSL' of type 'FileSystem' to resource group 'OgreInternal'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/RTShaderLib/HLSL_Cg' of type 'FileSystem' to resource group 'OgreInternal'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Media/Terrain/' of type 'FileSystem' to resource group 'OgreInternal'
09:33:12: Creating resource group Tests
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Tests/Media' of type 'FileSystem' to resource group 'Tests'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/programs/GLSL' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/programs/HLSL' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/programs/Cg' of type 'FileSystem' to resource group 'General'
09:33:12: Added resource location 'C:/Projects/NonTerra/OGRE/Samples/Media/materials/programs/HLSL_Cg' of type 'FileSystem' to resource group 'General'
09:33:12: Initialising resource group Essential
09:33:12: Parsing scripts for resource group Essential
09:33:12: Parsing script SdkTrays.material
09:33:12: Parsing script OgreProfiler.material
09:33:12: Parsing script SdkTrays.fontdef
09:33:12: Parsing script SdkTrays.overlay
09:33:12: Texture 'sdk_cursor.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_tray.png': Loading 1 faces(PF_A8B8G8R8,64x64x1) with 6 generated mipmaps from Image. Internal format is PF_A8B8G8R8,64x64x1.
09:33:12: Texture 'sdk_button_up.png': Loading 1 faces(PF_A8B8G8R8,128x32x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x32x1.
09:33:12: Texture 'sdk_text_box.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_mini_tray.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_track.png': Loading 1 faces(PF_A8B8G8R8,16x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,16x32x1.
09:33:12: Texture 'sdk_handle.png': Loading 1 faces(PF_A8B8G8R8,16x16x1) with 4 generated mipmaps from Image. Internal format is PF_A8B8G8R8,16x16x1.
09:33:12: Texture 'sdk_mini_text_box.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_label.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_separator.png': Loading 1 faces(PF_A8B8G8R8,64x16x1) with 6 generated mipmaps from Image. Internal format is PF_A8B8G8R8,64x16x1.
09:33:12: Texture 'sdk_logo.png': Loading 1 faces(PF_A8B8G8R8,128x64x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x64x1.
09:33:12: Texture 'sdk_shade.png': Loading 1 faces(PF_A8B8G8R8,64x48x1) with 6 generated mipmaps from Image. Internal format is PF_A8B8G8R8,64x48x1.
09:33:12: Texture 'sdk_frame.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_mini_text_box_over.png': Loading 1 faces(PF_A8B8G8R8,32x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,32x32x1.
09:33:12: Texture 'sdk_pulse.png': Loading 1 faces(PF_B8G8R8,8x1x1) with 3 generated mipmaps from Image. Internal format is PF_A8B8G8R8,8x1x1.
09:33:12: Finished parsing scripts for resource group Essential
09:33:12: Creating resources for group Essential
09:33:12: All done
09:33:12: Texture 'sdk_bands.png': Loading 1 faces(PF_A8B8G8R8,2x32x1) with 5 generated mipmaps from Image. Internal format is PF_A8B8G8R8,2x32x1.
09:33:12: Warning: Could not open 'C:\Users\markb\Documents\OGRE Sample Browser\cache.bin'
09:33:12: Parsing scripts for resource group BSPWorld
09:33:12: Font SdkTrays/Caption using texture size 256x256
09:33:12: Texture 'SdkTrays/CaptionTexture': Loading 1 faces(PF_BYTE_LA,256x256x1) with 0 custom mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:12: Font SdkTrays/Value using texture size 512x256
09:33:12: Texture 'SdkTrays/ValueTexture': Loading 1 faces(PF_BYTE_LA,512x256x1) with 0 custom mipmaps from Image. Internal format is PF_A8B8G8R8,512x256x1.
09:33:13: Parsing script scripts/clown.shader
09:33:13: Parsing script scripts/liquid_water.shader
09:33:13: Parsing script scripts/oalite.shader
09:33:13: Parsing script scripts/oanew.shader
09:33:13: Parsing script scripts/oasky.shader
09:33:13: Finished parsing scripts for resource group BSPWorld
09:33:13: Creating resources for group BSPWorld
09:33:13: All done
09:33:13: Parsing scripts for resource group General
09:33:13: Parsing script Examples.program
09:33:13: Parsing script Instancing.program
09:33:13: Parsing script StdQuad_vp.program
09:33:13: Parsing script deferred_post.program
09:33:13: Parsing script ShadowCaster.program
09:33:13: Parsing script pbr.program
09:33:13: Parsing script pbr_filament.program
09:33:13: Parsing script AdaptivePNTrianglesTessellation.material
09:33:13: Program 'Ogre/PNTrianglesTessellation/adaptive_tessellation_hs' is not supported: Cannot compile D3D11 high-level shader Ogre/PNTrianglesTessellation/adaptive_tessellation_hs Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\adaptive_tessellation.hlsl(148,67-71): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(208): Named constants have not been initialised, perhaps a compile error
09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(209): Named constants have not been initialised, perhaps a compile error
09:33:13: Program 'Ogre/PNTrianglesTessellation/adaptivePNTriangles_tessellation_vs' is not supported: Cannot compile D3D11 high-level shader Ogre/PNTrianglesTessellation/adaptivePNTriangles_tessellation_vs Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\adaptivePNTriangles_tessellation.hlsl(92,35-58): error X4577: Not all elements of SV_Position were written
C:\Projects\NonTerra\build\OGRE\bin\release\adaptivePNTriangles_tessellation.hlsl(92,35-58): error X4576: Output signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(258): Named constants have not been initialised, perhaps a compile error
09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(259): Named constants have not been initialised, perhaps a compile error
09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(260): Named constants have not been initialised, perhaps a compile error
09:33:13: Program 'Ogre/PNTrianglesTessellation/adaptivePNTriangles_tessellation_hs' is not supported: Cannot compile D3D11 high-level shader Ogre/PNTrianglesTessellation/adaptivePNTriangles_tessellation_hs Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\adaptivePNTriangles_tessellation.hlsl(196,67-71): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(274): Named constants have not been initialised, perhaps a compile error
09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(275): Named constants have not been initialised, perhaps a compile error
09:33:13: Program 'Ogre/PNTrianglesTessellation/adaptivePNTriangles_tessellation_ds' is not supported: Cannot compile D3D11 high-level shader Ogre/PNTrianglesTessellation/adaptivePNTriangles_tessellation_ds Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\adaptivePNTriangles_tessellation.hlsl(213,152-156): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(287): Named constants have not been initialised, perhaps a compile error
09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(288): Named constants have not been initialised, perhaps a compile error
09:33:13: Error: ScriptCompiler - invalid parameters in AdaptivePNTrianglesTessellation.material(289): Named constants have not been initialised, perhaps a compile error
09:33:13: Parsing script ASCII.material
09:33:14: Parsing script BlackAndWhite.material
09:33:14: Parsing script Bloom2.material
09:33:14: Parsing script CompositorDemo.material
09:33:14: Parsing script Compute.material
09:33:14: Parsing script DamagedHelmet.material
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(43): PBRSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(49): PBRSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(54): PBRSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(59): PBRSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(64): PBRSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(70): PBRLUTSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(75): PBRIBLSampler
09:33:14: Error: ScriptCompiler - reference to a non existing object in filament.material(81): PBRSampler
09:33:14: Parsing script DepthShadowmap.material
09:33:14: Parsing script Dither.material
09:33:14: Parsing script DOF.material
09:33:14: Parsing script DualQuaternion.material
09:33:14: Parsing script Embossed.material
09:33:14: Parsing script Examples-Advanced.material
09:33:14: Parsing script Examples-DynTex.material
09:33:14: Parsing script Examples-Water.material
09:33:14: Parsing script Examples.material
09:33:15: Parsing script facial.material
09:33:15: Parsing script FluidSim.material
09:33:15: Parsing script Glass.material
09:33:15: Parsing script Hair.material
09:33:15: Parsing script Halftone.material
09:33:15: Parsing script hdr.material
09:33:15: Parsing script HeatVision.material
09:33:15: Parsing script Hurt.material
09:33:15: Parsing script HWInstancing.material
09:33:15: Parsing script HW_VTFInstancing.material
09:33:15: Parsing script HW_VTF_LUTInstancing.material
09:33:15: Parsing script Invert.material
09:33:15: Parsing script Island.material
09:33:16: Error: ScriptCompiler - number expected in Island.material(246): extra parameters required by constant definition light_position
09:33:16: Error: ScriptCompiler - number expected in Island.material(329): extra parameters required by constant definition light_position
09:33:16: Program 'Ogre/Island/WaterPatchPS' is not supported: Cannot compile D3D11 high-level shader Ogre/Island/WaterPatchPS Errors:
error X3501: 'WaterPatchPS': entrypoint not found

09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(342): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(343): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(344): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(345): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - number expected in Island.material(346): extra parameters required by constant definition light_position
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(347): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(348): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(349): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(350): Named constants have not been initialised, perhaps a compile error
09:33:16: Error: ScriptCompiler - invalid parameters in Island.material(351): Named constants have not been initialised, perhaps a compile error
09:33:16: Parsing script IsoSurf.material
09:33:16: Parsing script Laplace.material
09:33:16: Parsing script LigthShafts.material
09:33:16: Parsing script MotionBlur.material
09:33:16: Parsing script MRTtest.material
09:33:16: Parsing script NightVision.material
09:33:16: Parsing script Ocean.material
09:33:16: Parsing script OffsetMapping.material
09:33:16: Parsing script Ogre.material
09:33:16: Parsing script OldMovie.material
09:33:16: Parsing script OldTV.material
09:33:16: Parsing script ParticleGS.material
09:33:17: Parsing script Penguin.material
09:33:17: Parsing script Posterize.material
09:33:17: Parsing script pssm.material
09:33:17: Parsing script RadialBlur.material
09:33:17: Parsing script RasterizationOrder.material
09:33:17: Parsing script RZR-002.material
09:33:17: Parsing script ShaderInstancing.material
09:33:17: Parsing script ShaderSystem.material
09:33:17: Parsing script SharpenEdges.material
09:33:17: Parsing script sibenik.material
09:33:17: Parsing script smoke.material
09:33:17: Parsing script Swizzle.material
09:33:17: Parsing script TerrainTessellation.material
09:33:17: Program 'Ogre/TerrainTessellation/HwTessellationPassThruVS' is not supported: Cannot compile D3D11 high-level shader Ogre/TerrainTessellation/HwTessellationPassThruVS Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(239,48-79): warning X3576: semantics in type overridden by variable/function or enclosing type
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(239,48-79): error X4577: Not all elements of SV_Position were written
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(239,48-79): error X4576: Output signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(97): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(98): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(99): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(100): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(101): Named constants have not been initialised, perhaps a compile error
09:33:17: Program 'Ogre/TerrainTessellation/TerrainScreenspaceLODHS' is not supported: Cannot compile D3D11 high-level shader Ogre/TerrainTessellation/TerrainScreenspaceLODHS Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(518,74): warning X3576: semantics in type overridden by variable/function or enclosing type
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(414,97-98): warning X3576: semantics in type overridden by variable/function or enclosing type
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(400,6-17): warning X3556: integer modulus may be much slower, try using uints if possible.
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(518,74): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(131): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(132): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(133): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(134): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(135): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(136): Named constants have not been initialised, perhaps a compile error
09:33:17: Program 'Ogre/TerrainTessellation/TerrainDisplaceDS' is not supported: Cannot compile D3D11 high-level shader Ogre/TerrainTessellation/TerrainDisplaceDS Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\TerrainTessellation.hlsl(599,53-63): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(148): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(149): Named constants have not been initialised, perhaps a compile error
09:33:17: Error: ScriptCompiler - invalid parameters in TerrainTessellation.material(150): Named constants have not been initialised, perhaps a compile error
09:33:17: Parsing script Tessellation.material
09:33:17: Program 'Ogre/Tessellation/TessellationHullProgramHLSL' is not supported: Cannot compile D3D11 high-level shader Ogre/Tessellation/TessellationHullProgramHLSL Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\Tessellation.hlsl(56,76-77): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:17: Error: ScriptCompiler - invalid parameters in Tessellation.material(28): Named constants have not been initialised, perhaps a compile error
09:33:17: Program 'Ogre/Tessellation/TessellationDomainProgramHLSL' is not supported: Cannot compile D3D11 high-level shader Ogre/Tessellation/TessellationDomainProgramHLSL Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\Tessellation.hlsl(97,53-56): error X4576: Input signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:17: Error: ScriptCompiler - invalid parameters in Tessellation.material(57): Named constants have not been initialised, perhaps a compile error
09:33:17: Program 'Ogre/Tessellation/VertexProgramHLSL' is not supported: Cannot compile D3D11 high-level shader Ogre/Tessellation/VertexProgramHLSL Errors:
C:\Projects\NonTerra\build\OGRE\bin\release\Tessellation.hlsl(35,26-58): error X4577: Not all elements of SV_Position were written
C:\Projects\NonTerra\build\OGRE\bin\release\Tessellation.hlsl(35,26-58): error X4576: Output signature parameter SV_Position (1-based Entry 0) type must be float32 and mask must be xyzw. 

09:33:17: Parsing script Tiling.material
09:33:18: Parsing script VarianceShadowmap.material
09:33:18: Parsing script VTFInstancing.material
09:33:18: Parsing script deferreddemo.material
09:33:18: Parsing script deferred_post.material
09:33:18: Parsing script deferred_post_minilight.material
09:33:18: Parsing script ShadowCaster.material
09:33:18: Parsing script ssao.material
09:33:18: Parsing script ROOM.material
09:33:18: Parsing script CreaseShading.material
09:33:18: Parsing script Crytek.material
09:33:18: Parsing script GBuffer.material
09:33:18: Parsing script HemisphereMC.material
09:33:18: Parsing script HorizonBased.material
09:33:18: Parsing script Modulate.material
09:33:18: Parsing script ShowDepth.material
09:33:18: Parsing script ShowNormals.material
09:33:19: Parsing script ShowViewPos.material
09:33:19: Parsing script SSAOPost.material
09:33:19: Parsing script UnsharpMask.material
09:33:19: Parsing script Volumetric.material
09:33:19: Parsing script triplanarReference.material
09:33:19: Parsing script CSMShadows.material
09:33:19: Program 'ShadowCasterVP' is not supported: Unable to compile Cg program ShadowCasterVP: The compile returned an error.
(0) : error C3001: no program defined

09:33:19: Error: ScriptCompiler - invalid parameters in CSMShadows.material(10): Named constants have not been initialised, perhaps a compile error
09:33:19: Parsing script glTF2_PBR.material
09:33:19: Parsing script filament.material
09:33:19: Parsing script RomanBath.material
09:33:19: Parsing script Sinbad.material
09:33:19: Parsing script emitted_emitter.particle
09:33:19: Texture 'flare.png': Loading 1 faces(PF_B8G8R8,256x256x1) with 8 generated mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:19: Parsing script Examples-Water.particle
09:33:19: Texture 'basic_droplet.png': Loading 1 faces(PF_B8G8R8,64x64x1) with 6 generated mipmaps from Image. Internal format is PF_A8B8G8R8,64x64x1.
09:33:19: Parsing script Examples.particle
09:33:19: Texture 'flaretrail.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'aureola.png': Loading 1 faces(PF_BYTE_LA,256x256x1) with 8 generated mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:19: Parsing script smoke.particle
09:33:19: Texture 'smoke.png': Loading 1 faces(PF_A8B8G8R8,256x256x1) with 8 generated mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:19: Parsing script Examples.compositor
09:33:19: Parsing script deferred.compositor
09:33:19: Parsing script ssao.compositor
09:33:19: Parsing script SSAO.compositor
09:33:19: Parsing script SSAOPost.compositor
09:33:19: Finished parsing scripts for resource group General
09:33:19: Creating resources for group General
09:33:19: All done
09:33:19: Parsing scripts for resource group OgreAutodetect
09:33:19: Finished parsing scripts for resource group OgreAutodetect
09:33:19: Creating resources for group OgreAutodetect
09:33:19: All done
09:33:19: Parsing scripts for resource group OgreInternal
09:33:19: Parsing script ShadowVolumeExtude.program
09:33:19: Parsing script DualQuaternionSkinning_Shadow.material
09:33:19: Parsing script HardwareSkinningShadow.material
09:33:19: Parsing script RTShaderSystem.material
09:33:19: Parsing script TriplanarTexturing.material
09:33:19: Finished parsing scripts for resource group OgreInternal
09:33:19: Creating resources for group OgreInternal
09:33:19: All done
09:33:19: Parsing scripts for resource group Tests
09:33:19: Parsing script glow.material
09:33:19: Finished parsing scripts for resource group Tests
09:33:19: Creating resources for group Tests
09:33:19: All done
09:33:19: Loading library .\DefaultSamples.dll
09:33:19: Installing plugin: DefaultSamplesPlugin
09:33:19: Plugin successfully installed
09:33:19: Loading library .\PlayPenTests.dll
09:33:19: Creating resource group VisualTestTransient
09:33:19: Installing plugin: PlaypenTestPlugin
09:33:19: Plugin successfully installed
09:33:19: Texture 'thumb_atomicc.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_bsp.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_bezier.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_bump.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_csm.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_camtrack.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_cel.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_char.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_comp.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_compute.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_cubemap.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_imgui.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_deferred.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_dualquaternionskinning.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_dyntex.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_terrain.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_facial.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_fresnel.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_particlegs.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_grass.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_isosurf.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_lightshafts.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_lighting.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_meshlod.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_newinstancing.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_ocean.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_pcz.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_tessellation.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_particles.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_pbr.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_visual_tests.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_ssao.png': Loading 1 faces(PF_B8G8R8,256x256x1) with 8 generated mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:19: Texture 'thumb_shadersystem.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_texturedfog.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_shadersystemmultilight.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_shadows.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_skelanim.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_skybox.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_skydome.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_skyplane.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_smoke.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_spheremap.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_texarray.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_texfx.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_trans.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_volumecsg.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_volumeterrain.png': Loading 1 faces(PF_B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_voltex.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:19: Texture 'thumb_water.png': Loading 1 faces(PF_A8B8G8R8,128x128x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x128x1.
09:33:27: Texture 'rockwall.tga': Loading 1 faces(PF_B8G8R8,256x256x1) with 8 generated mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:27: Mesh: Loading Sinbad.mesh.
09:33:27: Skeleton: Loading Sinbad.skeleton
09:33:27: Texture 'sinbad_body.tga': Loading 1 faces(PF_B8G8R8,512x512x1) with 9 generated mipmaps from Image. Internal format is PF_A8B8G8R8,512x512x1.
09:33:27: Texture 'sinbad_clothes.tga': Loading 1 faces(PF_B8G8R8,512x512x1) with 9 generated mipmaps from Image. Internal format is PF_A8B8G8R8,512x512x1.
09:33:27: Texture 'sinbad_sword.tga': Loading 1 faces(PF_B8G8R8,256x256x1) with 8 generated mipmaps from Image. Internal format is PF_A8B8G8R8,256x256x1.
09:33:27: Warning: the mesh 'Sinbad.mesh' includes vertices with more than 4 bone assignments. The lowest weighted assignments beyond this limit have been removed, so your animation may look slightly different. To eliminate this, reduce the number of bone assignments per vertex on your mesh to 4.
09:33:27: Creating swords
09:33:27: Mesh: Loading Sword.mesh.
09:33:27: Creating the chains
09:33:27: Texture 'ribbonband.png': Loading 1 faces(PF_B8G8R8,128x1x1) with 7 generated mipmaps from Image. Internal format is PF_A8B8G8R8,128x1x1.
09:33:37: Error: RTSS - creating GpuPrograms for pass 0 of 'Examples/LightRibbonTrail' failed
```

Thanks for the great engine and any help you can offer!

--- a/Media/ShadowVolume/OgreUnifiedShader.h
+++ b/Media/ShadowVolume/OgreUnifiedShader.h
@@ -23,6 +23,7 @@
 #define mat3 float3x3
 #define mat4 float4x4
 
+#define texture1D tex1D
 #define texture2D tex2D
 #define texture3D tex3D
 #define textureCube texCUBE

